### PR TITLE
Letters content updates 8 21 2017

### PR DESCRIPTION
**Pull request by rroueche**

### CODE REVIEW

**Comment by rroueche** [comment]
@peggygannon you're updates are in staging! and i've made 2 changes in this new PR here - check out the Files Changed section and lmk if you have any issues w/those updates.

@annekainicUSDS could you also review to make sure I didn't break anything? I'll merge once I get the OK from you both.

**Comment by peggygannon** [comment]
@rroueche Hello, it looks like some of my PL edits did not get input. If we could make the below tweaks, that would be great. Thanks!

When you download a letter, it will show this address. If this address is incorrect you may want to update it, but your letter will still be valid even with the incorrect address. [Update the address that appears on your letter.]

Proof of Service Letter
This card shows that you served honorably in the Armed Forces. This card can also be used in place of a VA identification card.

Service Verification Letter
This letter shows your branch of service, the date you started active duty, and the date you were discharged from active duty.

**Review by peggygannon — COMMENTED** [review]
@rroueche Hi, I put some requested tweaks in the note ^^. Let me know if you want me to edit directly into the PR. Happy to do so.

**Comment by rroueche** [comment]
ok updated, @annekainicUSDS does this look ok to merge to you?

File: src/js/letters/components/AddressSection.jsx
Comment (annekainicUSDS) [inline src/js/letters/components/AddressSection.jsx]: Did you mean to leave this as "EDIT"?

File: src/js/letters/components/AddressSection.jsx
Comment (annekainicUSDS) [inline src/js/letters/components/AddressSection.jsx]: I think you need a comma after the conditional statement: `If this address is incorrect, you may want to update it`.

**Review by annekainicUSDS — APPROVED** [review]
LGTM! Just a couple of minor comments.

File: src/js/letters/components/AddressSection.jsx
Comment (rroueche) [inline src/js/letters/components/AddressSection.jsx]: ha, yes - just a placeholder since it'll soon be a button

**Comment by rroueche** [comment]
this was merged about a week ago, and we reviewed with stakeholders. More content updates to come, but those are in separate issues.

**Pull request closed without merging**
